src/libraries/Microsoft.PowerFx.LanguageServerProtocol/Schemas/ControlToken.cs

```diff
@@ -0,0 +1,19 @@
+﻿// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
+using System;
+using System.Collections.Generic;
+using System.Text;
+using Microsoft.PowerFx.LanguageServerProtocol.Protocol;
+
+namespace Microsoft.PowerFx.LanguageServerProtocol.Schemas
+{
+    internal class ControlToken
+    {
+        // Control name
+        public string Name { get; set; }
+
+        // Representing the range of the control token
+        public LinkedList<List<uint>> EncodedTokenIndices { get; set; }
```

Comment:
Just call it range. EncodedTokenIndices is a concept of semantic tokens and shouldn't be a concept for this notification. We wouldn't decode these indices. we would just use them